cpp/marc_tools/marc_augmentor.cc

```diff
@@ -460,6 +538,11 @@ void ProcessAugmentorArgs(char **argv, std::vector<AugmentorDescriptor> * const
             ExtractCommandArgs(&argv, &tag, &subfield_code, &compiled_pattern, &field_or_subfield_contents);
             augmentors->emplace_back(AugmentorDescriptor::MakeInsertFieldIfAugmentor(tag, subfield_code, compiled_pattern,
                                                                                      field_or_subfield_contents));
+        } else if (std::strcmp(*argv, "--insert-field-if-regex") == 0) {
```

Comment:
There should also be a usage added for `--insert-field-if-regex` (`--add-subfield-if-matching` it has been added, but not for the other one)